dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerTest.java

```diff
@@ -1107,6 +1107,22 @@ public void testFindLinkageProblems_blockHoundClasses()
     }
   }
 
+  @Test
+  public void testFindLinkageProblems_unusedAvalonClassesFromCommonsLogging() throws Exception {
+    // The commons-logging has references to Avalon classes in optional dependencies.
+    // https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1871
+    // As we know they are not harmful, the default exclusion file skips them.
+    ClassPathResult classPathResult =
+        classPathBuilder.resolve(
+            ImmutableList.of(new DefaultArtifact("commons-logging:commons-logging:1.2")),
+            false,
+            DependencyMediation.MAVEN);
+    LinkageChecker linkageChecker = LinkageChecker.create(classPathResult.getClassPath());
+
+    ImmutableSet<LinkageProblem> problems = linkageChecker.findLinkageProblems();
+    Truth.assertThat(problems).isEmpty();
```

Comment:
nit: Perhaps we can make `Truth` a static import?

Reply:
Updated.